src/packet_analysis/Manager.h

```diff
@@ -126,6 +132,8 @@ class Manager : public plugin::ComponentManager<Tag, Component>
 		return pkt_filter;
 		}
 
+	uint64_t GetUnprocessedCount() const { return total_not_processed; }
```

Comment:
This is missing a documentation header :)

Reply:
Added